fix: selfhosted docs (#1340)
ory/docs: docs/self-hosted/operations/01_tracing.md

missing/wrong, please [create an issue](https://github.com/ory/docs/issues).

### Local setup

The [provided docker-compose file](https://github.com/ory/hydra/blob/master/quickstart-tracing.yml) in the project repository has
tracing configuration which you can use to play around with - just uncomment the desired tracing provider. We will use Jaeger as
an example.

Simply run

```sh
docker-compose -f quickstart.yml \
    -f quickstart-tracing.yml \
    up --build
```

Grab a coffee or stretch while you wait for everything to come up. You will then be able to navigate to the Jaeger UI which you
have exposed on port `16686` at http://localhost:16686/search. You can now start making requests and inspect traces!

As an example, here is a trace created by making a bad request to the `POST /clients` endpoint:

![OpenTracing and OpenCensus exemplary trace in Jaeger UI](../_static/sample_trace.png)

At a glance, you are able to see that:

- The request failed
- The request took ~80ms
- It resulted in a 409
- The hash comparison to validate the client's credentials took a whopping 70ms. Bcrypt is expensive!
- The various database operations performed

> _Note: in order to see spans around database interactions, you must be using a SQL backend (i.e. MySQL or Postgres)._

Reviewer: ```suggestion
:::note
 
To see spans around database interactions, you must be using a SQL backend, such as MySQL or Postgres.

:::
```
